src/rust-crypto/rust-crypto.ts

```diff
@@ -1182,6 +1166,26 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
         this.checkKeyBackupAndEnable();
     }
 
+    /**
+     * Implementation of {@link CryptoApi#importSecretsBundle}.
+     */
+    public async importSecretsBundle(
+        secrets: Parameters<NonNullable<CryptoApi["importSecretsBundle"]>>[0],
```

Comment:
I still think there's way too much type magic going on here. As a caller of this function, how am I meant to build one of these things?

Reply:
> As a caller of this function, how am I meant to build one of these things?

you shouldn't construct manually, only ever exported by the matching export function